apps/client/src/utils/index.js

```diff
@@ -37,35 +37,10 @@ export const getRestrictionByTypeName = (restrictions, typeName) =>
 
 // IMTR helper
 export const removeQuotes = (str) =>
-  typeof str === "string" ? str.replace(/['"]+/g, "") : "";
+  typeof str === "string" ? str.replace(/['"]+/g, "") : str;
```

Comment:
This does not make it easier to convert to typescript. I'm not sure why you even want to do this. It's also no a str anymore.

Reply:
I agree it's not TypeScript proof, I can do that as well. But I would expect that if I run `removeQuotes(12)` that it returns `12` or `"12"`, but not `""`

Reply:
I guess it should give a typescript error.
`12 is not a string.`